CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Unreleased
 
+[...]
+
+# v38.0.0 (28/07/2020)
```

Comment:
`v38.1.0` ?

Although since we an update and a NEW feature, should we make a major?

Reply:
Oops yes it's `38.1.0`
Major should be reserved for breaking changes, minor are here for new and retro-compatible features.